### PR TITLE
Support fabric 1.4

### PR DESCRIPTION
**Pull request by inzamam-iqbal**

### CODE REVIEW

**Comment by inzamam-iqbal** [comment]
I tested this PR (All three examples) with aidtech repo and it works. Before merging this PR we have to update the stable helm repo with the changes made to aidtech repo. I will make that PR for stable repo

File: examples/dev/nephos_config.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ core:
   # Comment out to make it work on your cluster
   cluster: minikube
   # ChartMuseum repository or directory holding charts
-  chart_repo: stable
+  chart_repo: aidtech
```
Comment (alexvicegrab) [inline examples/dev/nephos_config.yaml]: We should set this to "stable", as others won't have access to `aidtech` chart repo.

File: examples/prod/helm_values/hlf-ca/ca.yaml [hunk 2]
```diff
@@ -9,12 +9,12 @@ ingress:
   path: /
   hosts:
     # TODO: Change this to your Domain Name
-    - ca.nephos.local
+    - inzamam.tk
```
Comment (alexvicegrab) [inline examples/prod/helm_values/hlf-ca/ca.yaml]: We should not set this to a custom value, same in several other places.

File: examples/qa/nephos_config.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ core:
   # Comment out to make it work on your cluster
   cluster: minikube
   # ChartMuseum repository or directory holding charts
-  chart_repo: stable
+  chart_repo: aidtech
```
Comment (alexvicegrab) [inline examples/qa/nephos_config.yaml]: Same here

**Comment by alexvicegrab** [comment]
@inzamam-iqbal great, missed your comment. Assign me to the relevant PRs in the Helm Repo and I will approve them ASAP.

File: examples/qa/nephos_config.yaml (hunk 3 above)
Comment (alexvicegrab) [reply examples/qa/nephos_config.yaml]: Thanks

File: examples/dev/nephos_config.yaml (hunk 1 above)
Comment (alexvicegrab) [reply examples/dev/nephos_config.yaml]: Thanks

File: examples/prod/helm_values/hlf-ca/ca.yaml (hunk 2 above)
Comment (alexvicegrab) [reply examples/prod/helm_values/hlf-ca/ca.yaml]: We just need to change this one as well.

**Comment by alexvicegrab** [comment]
Thanks for latest update, just one more set of changes pending.

**Comment by alexvicegrab** [comment]
Perfect, thanks!

**Comment by inzamam-iqbal** [comment]
@alexvicegrab seems like those changes to helm repo are not required. Our current stable repo can support faric 1.4 without any changes. I tested all three example locally. So can we merge it?

**Comment by alexvicegrab** [comment]
If CI fails, you or I can restart it via Travis

**Pull request merged**